src/sentry/models/debugfile.py

```diff
@@ -74,15 +98,15 @@ def find_by_debug_ids(self, project, debug_ids, features=None):
 
         Returns a dict of debug files keyed by their debug identifier.
         """
-        features = frozenset(features) if features is not None else frozenset()
+        features: FrozenSet[str] = frozenset(features) if features is not None else frozenset()  # type: ignore
```

Comment:
Why type:ignore here?

May want to use the more abstract Collection here if it helps removing type:ignore

Reply:
mypy doesn't allow you to redefine a local name unless we add an `--allow-redifinition` argument.  I grepped the sentry code and found somewhere else that did exactly this with a `# type: ignore` so decided to copy it.  I think it is a bit pants to be fair, I think this is a very reasonable pattern.